### PR TITLE
Closest point on curve sample

### PR DESCRIPTION
**Pull request by Josh-Schifter**

### CODE REVIEW

**Review by tomdicarlo — APPROVED** [review]

File: src/common/InteractivePointMarker.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    ctx.fillStyle = "black";
+    ctx.lineWidth = 1;
+    ctx.arc(0, 0, 2, 0, Math.PI * 2);
+    ctx.fill();
+    ctx.stroke();
+  }
+}
+
+export class MoveSpacePointTool extends InputCollector {
+  public static toolId = "MovePointTool"; // <== Used to find flyover (tool name), description, and keyin from namespace tool registered with...see CoreTools.json for example...
```
Comment (williamkbentley) [inline src/common/InteractivePointMarker.ts:66]: I don't understand this comment. Seems half formed.

**Comment by williamkbentley** [comment]
Two items in "Problems" panel on UI file:

- '(props: {}, context: any): Component<{}, ClosestPointOnCurveState, any>' is deprecated
- Cannot find module 'common/InteractivePointMarker' or its corresponding type declarations.

**Comment by Josh-Schifter** [comment]
> Two items in "Problems" panel on UI file:
> 
> * '(props: {}, context: any): Component<{}, ClosestPointOnCurveState, any>' is deprecated
> * Cannot find module 'common/InteractivePointMarker' or its corresponding type declarations.

I fixed the first one.  Need @StefanRetief to look into the import problem.  I think we can merge this as is and let Stefan fix it up later.

**Review by williamkbentley — APPROVED** [review]
LGTM

**Pull request merged**